auto_editor/AddCommentsRule.py

```diff
@@ -113,5 +113,53 @@ def run_rule(self, project: StructuredProjectSource,
                     w_type = "DPCT"
                     return project
 
+                if w_type == "DPCT1009":
+                    del all_items[warning_first_line: warning_last_line + 1]
+                    comment_item = LineItem(prefix + "/*\n"
+                                                     + prefix + "CTA1009:" + count + ": CTA1009:count number: SYCL uses exceptions to report errors and does not use the error codes. \n"
+                                                     + prefix + "The original code was commented out and a warning string was inserted. You need to rewrite this code\n"
+                                                     + prefix + "*/\n")
+                    all_items.insert(warning_first_line, comment_item)
+                    w_type = "DPCT"
+                    return project
+
+                if w_type == "DPCT1010":
+                    del all_items[warning_first_line: warning_last_line + 1]
+                    comment_item = LineItem(prefix + "/*\n"
+                                                     + prefix + "CTA1010:" + count + ": Base on the experience, strongly recommended to ignore this warning.\n"
```

Comment:
Here and in other places (also existing recommendations): Instead of "Based on experience", maybe say "Based on analysed sample data"? We could even look at the relevant cases in sample data and say something like "In 10/10 cases in the sample data, this warning was ignored". I think it could also be helpful to contain parts of the original warning message if seems useful for the developer to evaluate the situation

Reply:
7/9 ignore for 1010, 9/9 for 1001/1000